src/loan/LoanTable/LoanTable.tsx

```diff
@@ -8,17 +8,18 @@ type Props = {
 }
 
 const statusTypes = [
-    {key: 'PENDING_EXTENSION', class: 'pending-approval',translation: 'Prórroga Pend.'},
+    {key: 'PENDING_EXTENSION', class: 'pending-approval', translation: 'Prórroga Pend.'},
     {key: 'APPROVED_EXTENSION', class: 'approved', translation: 'Prórroga Rech.'},
     {key: 'REJECTED_EXTENSION', class: 'rejected', translation: 'Prórroga Acp.'},
     {key: 'DELAYED', class: 'delayed', translation: 'Atrasado'},
     {key: 'WITHDRAWN', class: 'withdrawn', translation: 'Retirado'},
+    {key: 'READY_FOR_WITHDRAWAL', class: 'withdrawn', translation: 'Listo para ret.'},
```

Comment:
Please use a different class and also use another color for the `READY_FOR_WITHDRAWAL` status (`#cerulean` should be fine)